README.md

```diff
@@ -1,12 +1,16 @@
-# YAML Conf
-
-Yaml conf is the best way to talk about YAML.
-
 ```yaml
 ---
 yaml_conf: ON
 reasons:
   - I
   - ♥️
   - YAML
+when: 2020-12-12T12:12:12
+where:
+  city: Boulder
+  states: CA
+  country: USA
+links:
+  - https://onlineyamltools.com/edit-yaml
+  - https://yaml.org
```

Comment:
Are we sure we want to send people away from our website? Maybe we can link to a second page which redirects them to the website 🌮

Reply:
I thought it was ok to link to our friends website. We should now convince them to link to us.

Reply:
Soon we'll be able to link to our API!!!